qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.management;
+
+import static io.camunda.zeebe.it.management.ExportingEndpointIT.StableValuePredicate.hasStableValue;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.qa.util.actuator.ExportingActuator;
+import io.camunda.zeebe.qa.util.actuator.PartitionsActuator;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.cluster.ZeebeCluster;
+import io.zeebe.containers.exporter.DebugReceiver;
+import java.time.Duration;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Predicate;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+final class ExportingEndpointIT {
+
+  @Test
+  void shouldPauseExporting() {
+    final var exportedRecords = new CopyOnWriteArrayList<>();
+    try (final var receiver = new DebugReceiver(exportedRecords::add).start()) {
+      try (final var cluster =
+          ZeebeCluster.builder()
```

Comment:
:wrench: Now that there is only one configuration, can we use TestContainers extension?

Reply:
Good point :+1: Not sure if it's possible, we need some extra setup with the debug exporter and I'm never quite sure in what order resources are initialized by JUnit. I'm going to leave it as is for now.

Reply:
They're initialized from top-to-bottom. This is further guaranteed by Testcontainers if you annotate both things with a `Container` annotation. We could make the `DebugReceiver` a `Startable`, so it could be also managed by the Testcontainers extension. Then you would be guaranteed the ordering :+1: